mapper.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 var (
-	identifierRE = `[a-zA-Z_][a-zA-Z0-9_]+`
+	identifierRE = `[a-zA-Z_]+-?[a-zA-Z0-9_]+`
```

Comment:
Thanks!

`identifierRE` is used not only in the regex that matches statsd metric lines, but also for the other two ones `labelLineRE` and `metricNameRE`. Those are for matching Prometheus labels and Prometheus metrics, both of which may not contain dashes.

If you want to allow dashes in statsd metric names, you'll have to isolate that change to just the regex that matches those.